autogen/main/main.tf.tmpl

```diff
@@ -175,6 +175,13 @@ locals {
   cluster_zones    = sort(local.cluster_output_zones)
 
   cluster_name                               = local.cluster_output_name
+  {% if autopilot_cluster != true %}
+  // node pool ID is in the form project/location/cluster/name
```

Comment:
Based on logic below it looks like this should be the form?
```suggestion
  // node pool ID is in the form projects/<project-id>/locations/<location>/clusters/<cluster-name>/nodePools/<nodepool-name>
```

Reply:
Right thanks, updated